### PR TITLE
[SecurityBundle] add "service" option in remember_me firewall

### PR DESCRIPTION
**Pull request by Pchol**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no  
| Deprecations? | no
| Tests pass?   | yes 
| Fixed tickets | 
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

allow override remember me service

### CODE REVIEW

**Comment by chindit** [comment]
I think that's a nice idea, but why naming it `custom_service` and not just `service`  in config ?

**Comment by Pchol** [comment]
> I think that's a nice idea, but why naming it `custom_service` and not just `service` in config ?

thank you, i renamed to `service`, it's more accurate

**Comment by leofeyer** [comment]
We would need this for Contao, too (see #31078), so +1. 😉

**Comment by leofeyer** [comment]
> > I think that's a nice idea, but why naming it `custom_service` and not just `service` in config ?
> 
> thank you, i renamed to `service`, it's more accurate

I believe that the common name is `id`.

File: src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/RememberMeFactory.php [hunk 1]
```diff
@@ -130,6 +133,7 @@ public function addConfiguration(NodeDefinition $node)
 
         $builder
             ->scalarNode('secret')->isRequired()->cannotBeEmpty()->end()
+            ->scalarNode('service')->end()
```
Comment (fabpot) [inline src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/RememberMeFactory.php]: We would need an update to the XSD file to support the new option.

**Review by fabpot — CHANGES_REQUESTED** [review]

**Review by nicolas-grekas — APPROVED** [review]
(there is no xsd in the bundle)

**Comment by nicolas-grekas** [comment]
Thank you @Pchol.

**Pull request merged**